app/domain/authentication/authn_jwt/fetch_jwt_claims_to_validate.rb

```diff
@@ -26,13 +29,32 @@ def call
 
       private
 
-      OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
+      MANDATORY_CLAIMS = [EXP_CLAIM_NAME].freeze
+      OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
+
+      def validate_decoded_token_exists
+        if decoded_token.blank?
+          raise Errors::Authentication::AuthnJwt::MissingToken.new
+        end
```

Comment:
this can be one-lined:
```suggestion
        raise Errors::Authentication::AuthnJwt::MissingToken.new if decoded_token.blank?
```